common/config/rush/pnpmfile.js

```diff
@@ -40,6 +40,10 @@ function readPackage(packageJson, context) {
     packageJson.dependencies['typescript'] = '^2 || ^3';
   }
 
+  if (packageJson.name === 'monaco-editor-webpack-plugin') {
```

Comment:
Forgot to mention this yesterday, you should add a comment about why this is needed